### PR TITLE
Максакова Мария, 2536

### PR DESCRIPTION
**Pull request by Varteya**

### CODE REVIEW

**Comment by svasilinets** [comment]
Where is your apk?

**Comment by Varteya** [comment]
Не поняла, как добавлять файлы

**Comment by thevery** [comment]
FPS is ok, but final image is wrong
0
device-2014-11-09-051913

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 1]
```diff
@@ -10,14 +12,17 @@
...
-    int [][] field = null;
-    int width = 0;
-    int height = 0;
-    int scale = 4;
     final int MAX_COLOR = 10;
+    int[][] field = null;
+    int[][] field2;
+    Rect display;
+    final int width = 240;
+    final int height = 320;
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:23]: `public static final`

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 2]
```diff
@@ -100,14 +108,13 @@ void updateField() {
...
-    public void onDraw(Canvas canvas) {
-        for (int x=0; x<width; x++) {
-            for (int y=0; y<height; y++) {
-                Paint paint = new Paint();
-                paint.setColor(palette[field[x][y]]);
-                canvas.drawRect(x*scale, y*scale, (x+1)*scale, (y+1)*scale, paint);
+
+    public void updateCanvas(Canvas canvas) {
+        for (int x = 0; x < width; x++) {
+            for (int y = 0; y < height; y++) {
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:114]: you have the same loop in `updateField`

**Pull request closed without merging**